### ISSUE TITLE
[Screen Reader - Adaptive Cards iOS - V1.0] Label is not associated with “your name” text field present on “InputForm” page. 

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7003]
https://microsoft.visualstudio.com/OS/_workitems/edit/29462554

**Comment by carlos-zamora** [comment]
This bug cannot be fixed in v1.0. The author must use v1.3 and update the card accordingly (see [this sample card](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.3/Scenarios/InputFormWithLabels.json)). As such, closing the bug.
